boost-maven/boost-boms/boost-ee7-apis-bom/pom.xml

```diff
@@ -24,6 +24,11 @@
 				<artifactId>javax.ws.rs-api</artifactId>
 				<version>2.0</version>
 			</dependency>
+			<dependency>
+			    <groupId>org.apache.derby</groupId>
+			    <artifactId>derby</artifactId>
+			    <version>10.14.2.0</version>
+			</dependency>
 		</dependencies>            
```

Comment:
Why do we want to put this in the BOM?   It's not needed to compile against.    Though it'd be kind of nice to see these internal dependencies in a POM somewhere instead of buried in Java source...not sure we have a good way to do that.